website/docs/quickstart_guide/docker.md

```diff
@@ -41,13 +43,17 @@ They are not suitable for most production use-cases because they keep all data i
    `ferretdb` runs FerretDB.
 
 2. Start services with `docker compose up -d`.
+   To be sure that you've fetched the latest version you can precede the command with `docker compose pull`.
 
 3. If you have `mongosh` installed, just run it to connect to FerretDB.
```

Comment:
"just run it to connect to FerretDB" doesn't seem enough anymore as we need to mention username, password and authentication method.